corehq/apps/sms/templates/sms/chat.html

```diff
@@ -25,6 +25,7 @@
         <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
         <script src="{% static 'hqstyle/js/jquery/jquery-migrate-1.2.1.min.js' %}"></script>
         <script type="text/javascript" src="{% static 'knockout-2.3.0-legacy/knockout.js' %}"></script>
+        <script src="{% static 'style/js/hq.helpers.js' %}"></script>
```

Comment:
i agree in a perfect world that it'd be great to remove all calls to `koApplyBindings` but personally feel like this is an exception. `chat.html` doesn't really use any HQ things besides that one file. also `ko.applyBindings` will always exist; it's not like we could delete a file or something if we could switch all references. no test coverage makes the cost of p2 seem like a pretty high risk